### PR TITLE
[READY](?) Gives Warden/Hos/Det donuts healing powers with working code

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
The Hos/Warden and detective now know the true power that is Donuts and how there grate in ever way. Even without coffee

[why]
There sec. Like no really They are. Not the lawyer tho hes a Muffin fella

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
Its tested and such Locally and all
40 -> 42 -> 44 ect
https://cdn.discordapp.com/attachments/499347891161137172/501852198699794432/unknown.png

**Comment by Poojawa** [comment]
lol, this is one of those oversights so 👍

**Comment by Iamgoofball** [comment]
Did you test this? You didn't test this.

File: code/modules/food_and_drinks/food/snacks_pastry.dm [hunk 1]
```diff
@@ -28,7 +28,7 @@
 	if(last_check_time + 50 < world.time)
 		if(ishuman(M))
 			var/mob/living/carbon/human/H = M
-			if(H.mind && H.mind.assigned_role == "Security Officer" && !H.has_trait(TRAIT_AGEUSIA))
+			if(H.mind && H.mind.assigned_role == "Security Officer, Warden, Head of Security, Detective" && !H.has_trait(TRAIT_AGEUSIA))
```
Comment (Iamgoofball) [inline code/modules/food_and_drinks/food/snacks_pastry.dm]: This is how I know you didn't test it.

Comment (Trilbyspaceclone) [reply code/modules/food_and_drinks/food/snacks_pastry.dm]: What, I tested it and ate a donut at 40 HP,  non-frosted 42 44 46 HP each bite. That means it healed the one area that was harmed 2 hp at a time, at once. Same happend when I ate a donut as a Sec officer at the same even HP level.

Comment (silicons) [reply code/modules/food_and_drinks/food/snacks_pastry.dm]: this isn't how it works
H.mind.assigned_role == "Security Officer" || H.mind.assigned_role == "Warden" etc etc
or H.mind.assigned_role in list("Security Officer", "Warden", etc etc)
not like this.

**Comment by Trilbyspaceclone** [comment]
Proof that it works better now somehow?
https://cdn.discordapp.com/attachments/455477249299185704/502603778353266689/unknown.png

**Review by silicons — APPROVED** [review]

**Pull request merged**